server/service/src/programs/patient/update_patient.rs

```diff
@@ -93,7 +96,7 @@ pub(crate) fn update_patient(
                 .rows
                 .pop()
                 .ok_or(UpdatePatientError::InternalError(
-                    "Can't find the just inserted patient".to_string(),
+                    "Can't find the newly created patient".to_string(),
```

Comment:
Thanks!

Reply:
wrong message lol. I've changed it again! xD